.circleci/config.yml

```diff
@@ -181,6 +201,21 @@ workflows:
     jobs:
       - format_flutter
       - analyze_flutter
+      - pub_analysis:
+          matrix:
+            parameters:
+              plugin:
+                [
+                  "amplify_authenticator",
+                  # TODO: enable other packages once pub score is addressed
```

Comment:
`amplify_authenticator` is the only package with a max score right now.

Note: 120 is considered a max score for packages that don't support web, even though adding web support would bump the score to 130. It is still considered a "max" by pana,

Reply:
It seems like the other packages lose points for, among other things, a dev dependency on unpublished package amplify_test. Any ideas for how to resolve that? Without fixing that, it seems we will either a) have to use only in authenticator b) change the exit-code-threshold to 10 (I think?) to basically ignore that loss of points.

Reply:
I don't know off the top of my head. If we cannot figure out how to address that, we could set the threshold to 10, or set different threshold per package.

Reply:
ok, makes sense. For this PR, I would either

a) merge it like this and figure it out when we add other plugins
b) do this now for any packages that can be published with threshold of 10 or 0.

Reply:
I updated the PR to use a different threshold for each plugin. The one quirk with this is that the threshold is now in the CI job name. That means we will have to change the list of required jobs each time we adjust the threshold. I think thats fine though as it shouldn't be changed frequently.